frontend/src/HomeGuest.js

```diff
@@ -0,0 +1,107 @@
+            <li className="text-white mx-4 font-semibold">
+              <a className="hover:text-gray-300"><Link to="/app">Lokalizacja klubów</Link></a>
+            </li>
+            <li className="text-white mx-4 font-semibold">
+              <a href="#" className="hover:text-gray-300">Spis klubów</a>
+            </li>
+            <li className="text-white mx-4 font-semibold">
+              <a className="hover:text-gray-300"><Link to="/auth">Logowanie / Rejestracja</Link></a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <Banner backgroundImage={data[2]} />
+
+    {/* About Us */}
+    <section className="flex flex-col md:flex-row items-center justify-center text-center bg-[#353230] text-white p-6 md:p-32">
+      <div className="md:flex-1 md:mr-6">
+        <h2 className="text-3xl font-bold mb-4">Czym jest kibolApp - ABOUT</h2>
+          <p className="text-lg mb-6">
+            KibolApp to innowacyjna strona stworzona dla pasjonatów piłki nożnej i kibiców, którzy chcą poznać kulisy relacji między klubami.
+          </p>
+          <p className="text-lg mb-6">
+            Aplikacja umożliwia śledzenie lokalizacji klubów, a także dostarcza informacje na temat przyjaźni i konfliktów między różnymi drużynami.
+          </p>
+      </div>
+      <div className="md:flex-1 max-w-md mx-auto">
+        <img src={data[3]} alt="About Us" className="w-full h-auto rounded-lg" />
+      </div>
+    </section>
+
+    {/* Contact Us */}
+    <section className="bg-custom-sand py-16">
+      <div className="flex justify-center">
+        <div className="max-w-2xl w-full">
+          <h3 className="text-white text-3xl text-center mb-10 font-bold">Masz pytania? <span className="text-custom-brown uppercase font-bold">Skontaktuj się z nami</span></h3>
+            <form className="flex flex-col items-center">
+              <div className="flex w-full mb-6">
+                <input type="text" placeholder="Name" className="flex-1 bg-transparent border-b border-black text-black px-2 py-1 outline-none placeholder-black placeholder-opacity-50 mr-4" />
+                <input type="email" placeholder="E-mail" className="flex-1 bg-transparent border-b border-black text-black px-2 py-1 outline-none placeholder-black placeholder-opacity-50" />
+              </div>
+              <div className="w-full mb-6">
+                <textarea placeholder="Napisz swoją wiadmość tutaj..." className="w-full bg-transparent border-b border-black text-black px-2 py-1 outline-none placeholder-black placeholder-opacity-50 h-24 resize-none"></textarea>
+              </div>
+              <div className="w-full flex justify-center">
+                <input type="submit" value="Wyślij!" className="bg-custom-olive text-custom-sand uppercase tracking-widest cursor-pointer font-bold px-4 py-2" />
+              </div>
+            </form>
+        </div>
+      </div>
+    </section>
+    </>
+
+  );
+};
+
+export default HomeGuest;
```

Comment:
Later we will need a new header for logged in account and Guest. But this is job for our frontend guys